manifests/init.pp

```diff
@@ -316,7 +316,8 @@
     require               => Class['pulpcore'],
   }
 
-  if $puppet {
+  # classes is set by Kafo
+  if pick($puppet, defined(Class['puppet']) or 'puppet' in lookup('classes', Array, undef, [])) {
```

Comment:
These types of changes always feel ugly to me -- where we are balancing building a puppet module and wrapping that with kafo and then backing in "kafo" knowledge to a module. Anyway, does not block just stating it as it more tightly links functionality to Kafo design.